cloudify_rest_client/secrets.py

```diff
@@ -75,20 +75,30 @@ def provider_name(self):
         """
         return self.get('provider_name')
 
+    @property
+    def provider_options(self):
+        """
+        :return: the secret's provider options
```

Comment:
plz more info about this, like an example or something. "provider options" is just not clear enough